vi/news/_posts/2015-04-13-ruby-2-2-2-released.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: news_post
+title: "Ruby 2.2.2 Released"
+author: "nagachika"
+translator: "Dôn Đinh"
+date: 2015-04-13 12:00:00 +0000
+lang: vi
+---
+
+Chúng tôi vui mừng thông báo về việc phát hành Ruby 2.2.2. Đây là một phiên bản nhỏ của seri 2.2.
+
+Phiên bản này đã sửa chữa lỗi bảo mật do lỗ hổng xác minh cho một hostname của OpenSSL extension.
+
+* [CVE-2015-1855: Ruby OpenSSL Hostname Verification](https://www.ruby-lang.org/en/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```

Comment:
The title should use Vietnamese: `Xác nhận Hostname Ruby OpenSSL` and should link to trasnlation: `https://www.ruby-lang.org/vi/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/`:

```
* [CVE-2015-1855: Xác nhận Hostname Ruby OpenSSL](https://www.ruby-lang.org/vi/news/2015/04/13/ruby-openssl-hostname-matching-vulnerability/)
```

:smile: